src/plugins/intel_gpu/tests/unit/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -1212,7 +1212,7 @@ class fully_connected_gpu_tests: public ::testing::Test {
             input_layout("input", input_mem->get_layout()),
             data("weights", weights_mem),
             data("scale", scale_mem),
-            fully_connected("fc_prim", input_info("input"), "weights", "", "scale", "", data_types::f32, padding(), 2, 2)
+            fully_connected("fc_prim", input_info("input"), "weights", "", "scale", "", data_types::f16, padding(), 2, 2)
```

Comment:
What is the purpose of changing this test?

Reply:
This change is because behavior of FC's layout calculation has been updated. Previously, the input data type was propagated as is, regardless of output data type specified in prim. However, as Convert is fused into FC, the output data type has been updated to have priority over input data type, so the test case must also be updated to specify the expected actual output data type.